### PR TITLE
update descriptions and examples in swagger.yml

### PR DESCRIPTION
**Pull request by VivianAllen**
### What

Describe what you have changed and why.

### How to review

Describe the steps required to test the changes.

### Who can review

Describe who worked on the changes, so that other people can review.

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -99,24 +100,32 @@ paths:
...
-      descriptions: Returns a list of the maximum values in each ckmean cluster.
+      description: |
+        Uses ckmeans algorithm to estimate natural breakpoints in the given census data category (*cat* parameter), 
+        filtered to only rows for the specified geography type (*geotype* parameter). Returns a list of the maximum 
+        values (i.e. the upper breakpoint) in each ckmean cluster.
       parameters:
         - in: path
           name: year
-          description: census year
+          description: Census year (currently only 2011 available).
```
Comment (stmuk) [inline swagger.yaml:110]: If we get synthetic data for 2021 quickly this might not be true in a week or two!

File: swagger.yaml [hunk 2]
```diff
@@ -149,28 +167,39 @@ paths:
...
-      descriptions: Returns a list of the maximum values in each ckmean cluster.
+      description: |
+        Uses ckmeans algorithm to estimate natural breakpoints in the ratio between two given census data category 
+        (*cat1* paramter / *cat2* parameter), filtered to only rows for the specified geography type (*geotype* parameter). 
+        Returns a list of the maximum values (i.e. the upper breakpoint) in each ckmean cluster.
       parameters:
         - in: path
           name: year
-          description: census year
+          description: Census year (currently only 2011 available).
```
Comment (stmuk) [inline swagger.yaml:177]: Ditto

File: swagger.yaml [hunk 3]
```diff
@@ -206,42 +243,73 @@ paths:
...
```
Comment (stmuk) [inline swagger.yaml:246]: ditto

**Review by wavemechanics — APPROVED** [review]

File: swagger.yaml [hunk 4]
```diff
@@ -17,16 +17,17 @@ paths:
...
           required: true
           schema:
             type: integer
         - in: query
           name: filtertotals
+          description: Use filtertotals=true if you want to have 'totals' categories seperated from other categories in the response (see Examples).
```
Comment (stmuk) [inline swagger.yaml:30]: seperated -> separated (caught by spell check since I can't spell myself!)

**Pull request merged**